rust_src/src/multibyte.rs

```diff
@@ -165,16 +166,17 @@ impl Codepoint {
     }
 
     // Equivalent to UNIBYTE_TO_CHAR
-    pub fn unibyte_to_char(self) -> Codepoint {
-        match self.is_ascii() {
-            true => self,
-            false => Codepoint::from_raw(self.0 as u8),
+    pub fn unibyte_to_char(self) -> Self {
+        if self.is_ascii() {
```

Comment:
What is wrong with using `match` on the bool like before? Is it poor Rust style or was this a personal choice?

Reply:
Clippy flags it. Not important of course, but matching on bools does seem like a strange style.